### PR TITLE
compute_val_loss parser action to store_true

### PR DESCRIPTION
**Pull request by mrtgenet**
Either the argument should be named "--no-val-loss" or something like this and have a store_false action, or be named "--compute-val-loss" and have a store_true action. Second option is more coherent with the fact that providing a validation set is optional.

See [this issue](https://github.com/fizyr/keras-retinanet/issues/976)

### CODE REVIEW

**Comment by hgaiser** [comment]
Thank you! But could you change the PR to pull to fizyr/keras-retinanet? :P

**Comment by mrtgenet** [comment]
@hgaiser oopsie, my bad, I change this
